digital_land/expectations/issue.py

```diff
@@ -19,6 +19,8 @@ def issue_factory(scope):
         "row-group": RowGroupIssue,
         "row": RowIssue,
         "value": ValueIssue,
+        "duplicate_reference": DuplicateReferenceIssue,
```

Comment:
I'm not sure if this should be a "scope" or not? I thought scope was for the "level" of what's wrong, so it would be a "row" scope, and the DuplicateReferenceIssue and InvalidReferenceIssue classes inherit from RowIssue. I'll tag @eveleighoj for his input.

Reply:
No I agree, the scope of an issue should affect what details need to be provided. the scope I think for this should be a row if the issue is specific to a row or a row group if it's multiple rows.